### PR TITLE
fix: message include '@' can be send now

### PR DESCRIPTION
**Pull request by fengjingyu1995**

### CODE REVIEW

**Comment by Anduin2017** [comment]
Hi, thanks for your contribution!

Seems your changes can not be build.

```log
ng lint --type-check && tsc

Linting "kahla"...

ERROR: /home/vsts/work/1/s/src/app/Services/MessageService.ts:347:61 - missing whitespace
```

Could you please fix it?

File: src/app/Services/MessageService.ts [hunk 1]
```diff
@@ -344,18 +344,20 @@ export class MessageService {
...
-            const matchedUsers = [];
-            this.conversation.users.forEach((value: UserGroupRelation) => {
-                if (!getMessage && value.user.nickName.toLowerCase().replace(/ /g, '').includes(nickName.toLowerCase())) {
-                    matchedUsers.push(value.user);
-                } else if (getMessage && value.user.nickName.toLowerCase().replace(/ /g, '') === nickName.toLowerCase()) {
-                    matchedUsers.push(value.user);
-                }
-            });
-            return matchedUsers;
+        if (typeof this.conversation.users !== 'undefined' ){
```
Comment (Anduin2017) [inline src/app/Services/MessageService.ts]: Wrong position of white space.

**Review by Anduin2017 — APPROVED** [review]

**Review by hv0905 — APPROVED** [review]

**Review by Anduin2017 — APPROVED** [review]

**Comment by Anduin2017** [comment]
Thanks for your contribution. We will merge it once it passes CI.

**Comment by Anduin2017** [comment]
Your changes have been merged. It will be deployed to our [staging env](https://staging.kahla.app) in about 15-20 mins.

**Pull request merged**